Backend/WebAPI/Controllers/ResetController.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using ServiceLayer.Services;
+using ManagerLayer.UserManagement;
+using DataAccessLayer.Models;
+
+
+namespace WebAPI.Controllers
+{
+    public class ResetController : ApiController
+    {
+
+        //After the user fills in the field with email, this action gets called
+        [HttpPost]
+        [ActionName("SendEmail")]
+        public void SendResetEmail(string email)
+        {
+            ResetService rs = new ResetService();
```

Comment:
Services should not be called by the API routes, order of operations should go API Routes call the appropriate method in the Manager Layer, Manager Layer is where the Database Context is initalized so that we are able to roll back any changes if anything goes wrong, Manager Layer is in charge of calling all necessary services, services call the repo which make the database changes then return a value. This continues until all necessary services are called by the managerLayer, in which if everything worked out the manager layer will be in charge of db.saveChanges().